src/ActionsMenuToggle.scss

```diff
@@ -0,0 +1,10 @@
+.ActionsMenu__toggle-button {
```

Comment:
ActionsMenu toggle is positioned at the last column for the Action header, and first vertically align it using align-self: center, and make it start from the middle of the last column by setting justify-self: center

`
.ActionsMenu__toggle-button {
    grid-column-start: 3;
    grid-column-end: 4;
    align-self: center;
    justify-self: center;
    width: 1em;
}
`